crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -540,363 +574,166 @@ module bridge::bridge {
+    #[test_only]
+    public fun assert_not_paused(bridge_inner: &BridgeInner, error: u64) {
+        assert!(!bridge_inner.paused, error);
+    }
 
-    #[test]
-    fun test_get_token_transfer_action_status() {
-        let mut scenario = test_scenario::begin(@0x0);
-        let ctx = test_scenario::ctx(&mut scenario);
-        let chain_id = chain_ids::sui_testnet();
-        let mut bridge = new_for_testing(ctx, chain_id);
-        let coin = coin::mint_for_testing<ETH>(12345, ctx);
+    #[test_only]
+    public fun test_get_current_seq_num_and_increment(
+        bridge_inner: &mut BridgeInner, 
+        msg_type: u8,
+    ): u64 {
+        get_current_seq_num_and_increment(bridge_inner, msg_type)
+    }
 
-        // Test when pending
-        let message = message::create_token_bridge_message(
-            chain_ids::sui_testnet(), // source chain
-            10, // seq_num
-            address::to_bytes(ctx.sender()), // sender address
-            chain_ids::eth_sepolia(), // target_chain
-            hex::decode(b"00000000000000000000000000000000000000c8"), // target_address
-            1u8, // token_type
-            balance::value(coin::balance(&coin))
-        );
+    #[test_only]
+    public fun test_execute_update_bridge_limit(
+        inner: &mut BridgeInner, 
+        payload: UpdateBridgeLimit,
+    ) {
+        execute_update_bridge_limit(inner, payload)
+    }
 
-        let key = message::key(&message);
-        linked_table::push_back(&mut load_inner_mut(&mut bridge).token_transfer_records, key, BridgeRecord {
-            message,
-            verified_signatures: option::none(),
-            claimed: false,
-        });
-        assert_eq(get_token_transfer_action_status(&mut bridge, chain_id, 10), TRANSFER_STATUS_PENDING);
+    #[test_only]
+    public fun test_execute_update_asset_price(
+        inner: &mut BridgeInner, 
+        payload: UpdateAssetPrice,
+    ) {
+        execute_update_asset_price(inner, payload)
+    }
 
-        // Test when ready for claim
-        let message = message::create_token_bridge_message(
-            chain_ids::sui_testnet(), // source chain
-            11, // seq_num
-            address::to_bytes(ctx.sender()), // sender address
-            chain_ids::eth_sepolia(), // target_chain
-            hex::decode(b"00000000000000000000000000000000000000c8"), // target_address
-            1u8, // token_type
-            balance::value(coin::balance(&coin))
-        );
-        let key = message::key(&message);
-        linked_table::push_back(&mut load_inner_mut(&mut bridge).token_transfer_records, key, BridgeRecord {
-            message,
-            verified_signatures: option::some(vector[]),
-            claimed: false,
-        });
-        assert_eq(get_token_transfer_action_status(&mut bridge, chain_id, 11), TRANSFER_STATUS_APPROVED);
+    #[test_only]
+    public fun transfer_status_pending(): u8 {
+        TRANSFER_STATUS_PENDING
+    }
```

Comment:
can we make these const public instead?

Reply:
unfortunately there is no public const concept in move, all const are private to the module